source/ways-of-working/guides/environments.html.md.erb

```diff
@@ -0,0 +1,22 @@
+---
+title: Environments
+last_reviewed_on: 2021-11-01
+review_in: 6 months
+weight: 10
+---
+
+# <%= current_page.data.title %>
+
+
+
+| Shared Services Environment | CNP Environment | Use case &nbsp; | Path to live | Additional Information &nbsp; &nbsp; |
+| - | - | - | - | - | - |
+| Prod            | Prod       | Live services | Yes |  |
+| staging         | AAT        | Automated Acceptance Testing before moving to Prod | Yes |  |
+| Demo            | Demo       | External demonstrations (no VPN required) | No | Used for demonstrating to external stakeholders |
+| ITHC            | ITHC       | IT Health Check, used for penetration testing | No |  |
+| Test            | Perftest   | Performance testing | No | |
+| Dev             | Preview    | Development of software changes | Yes | <ul> <li>Destroyed nightly</li> <li>Pull requests are deployed</li>​ <br> <li>Automatically tested with functional tests​</li> <br> <li>Application points to AAT instance for dependant services</li> </ul> |
```

Comment:
Maybe not say 'development' here - we're trying to move teams away from using it as a personal dev environment.

```suggestion
| Dev             | Preview    | Pre-merge automated testing and Pull Request checks | Yes | <ul> <li>Destroyed nightly</li> <li>Pull requests are deployed</li>​ <br> <li>Automatically tested with functional tests​</li> <br> <li>Application points to AAT instance for dependant services</li> </ul> |
```